nemoguardrails/guardrails/tool_rail_action.py

```
@@ -55,14 +56,12 @@ def __init__(self, tracer: Optional["Tracer"] = None) -> None:
         """Run *check* inside an action span, converting any error into a block.
 
-        Mirrors ``RailAction``'s error contract: an unexpected exception is recorded
-        on the span and returned as ``RailResult(is_safe=False, ...)`` so a malformed
+        Shares the engine's fail-closed contract with every other rail through
+        :func:`~nemoguardrails.guardrails.rail_guard.rail_error_result`, so a malformed
         input or a rail bug fails closed rather than crashing the request.
         """
         with action_span(self._tracer, self.action_name) as span:
             try:
                 return check()
             except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)